pgmoon/init.moon

```diff
@@ -32,6 +32,13 @@ flipped = (t) ->
     t[t[key]] = key
   t
 
+int8_serialization = ->
+  -- Lua prior to v5.3 does not support 64-bit integers
+  if 9223372036854775807 - 1 == 9223372036854775807  -- Lua <= 5.2
```

Comment:
Alright, thanks. I think a future project would be to get CI running on 5.2 and 5.3. I don't remember right now why it's disabled.